kai/config.toml

```diff
@@ -4,28 +4,32 @@ log_dir = "$pwd/logs"
 demo_mode = false
 trace_enabled = true
 
-# **Postgresql incident store**
-# [incident_store]
-# provider = "postgresql"
+solution_consumers = ["diff_only", "llm_summary"]
```

Comment:
Might be nice to either add a little detail on what it means here or add a link to some kind of explanation